src/test/java/org/sagebionetworks/bridge/workerPlatform/bridge/BridgeHelperTest.java

```diff
@@ -679,36 +679,7 @@ public void redriveUpload_timeout() throws Exception {
     }
 
     @Test
-    public void testGetStudyParticipantsForAppStudyId() throws IOException, IllegalAccessException, InterruptedException {
```

Comment:
Double-check you didn't accidentally delete something.

Reply:
Yes, this diff is confusing because I renamed a test to reflect the method it was calling on the worker (testGetSponsoredStudiesForApp) AND removed a utility method since there is only one test of that method now, for a given studyId (and not orgId).